styles/main.css

```diff
@@ -0,0 +1,96 @@
+* {
+  margin: 0;
+  padding-top: 0;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: "Inter", sans-serif;
+  font-size: 10px;
+}
+
+header {
+  display: flex;
+  width: 100vw;
+  position: fixed;
+  top: 0;
+  min-height: 8vh;
+  align-items: center;
+  justify-content: space-between;
+}
```

Comment:
- [ ] [OPTIONAL] You should try as much as possible to stick to the Figma design including the image and text colors. For this case, I suggest to change the color of the logo and humberger menu to match the Figma design.
- [ ] [OPTIONAL] In addition, the logo should be customized. Since this is your portifolio, you may consider using your name as the logo ☺️.